lms/djangoapps/instructor_task/tests/test_models.py

```diff
@@ -98,9 +108,9 @@ def create_report_store(self):
         Create and return a DjangoStorageReportStore using the old
         S3ReportStore configuration.
         """
-        connection = boto.connect_s3()
-        connection.create_bucket(settings.GRADES_DOWNLOAD['BUCKET'])
-        return ReportStore.from_config(config_name='GRADES_DOWNLOAD')
+        with patch.object(ReportStore, 'from_config', return_value=MockReportStore()):
+            self.mocked_connection.create_bucket(settings.GRADES_DOWNLOAD['BUCKET'])
+            return ReportStore.from_config(config_name='GRADES_DOWNLOAD')
```

Comment:
With `ReportStore.from_config()` mocked, there's really no application code being tested here anymore.  Is there a good way to test that method, or should this whole test case be removed?